cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetadataStoreDataset.java

```diff
@@ -192,6 +212,23 @@ public Key build() {
         return new Key(key);
       }
     }
+
+    /**
+     * Splits {@link Key}s.
+     */
+    public static final class Splitter {
+      public static List<byte[]> split(byte[] key) {
```

Comment:
Why a static method inside a static inner class? Can't it be just a private method? Also, never good to have public inner class. If it is truly a public util method, move it into `Bytes`.